packages/atomic/src/components/commerce/facets/atomic-commerce-facets/atomic-commerce-facets.tsx

```diff
@@ -0,0 +1,126 @@
+import {NumberValue, Schema} from '@coveo/bueno';
+import {
+  RegularFacet,
+  CategoryFacet,
+  DateFacet,
+  NumericFacet,
+  FacetGeneratorState,
+  buildProductListing,
+  buildSearch,
+  FacetGenerator,
+} from '@coveo/headless/commerce';
+import {Component, h, Element, Host, State, Prop} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {CommerceBindings as Bindings} from '../../atomic-commerce-interface/atomic-commerce-interface';
+
+/**
+ * The `atomic-commerce-facets` component automatically renders commerce facets based on the search query response.
```

Comment:
```suggestion
 * The `atomic-commerce-facets` component automatically renders commerce facets based on the commerce API response
```

The `search` keyword is misleading here, since facets are also available for product-listing